docs/source/index.mdx

```diff
@@ -1,138 +1,68 @@
 ---
 title: Introduction to Apollo Federation
-description: Combine GraphQL APIs into a unified supergraph
+subtitle: Learn how to combine your GraphQL APIs into a unified supergraph
+description: Learn how Apollo Federation can help you combine your GraphQL APIs into a unified, federated supergraph
```

Comment:
This serves as an SEO meta description. There may be a few more keywords to add here.